src/routes/router.js

```diff
@@ -58,6 +59,7 @@ export default Router.extend({
     'scratchpad': 'scratchpad',
     'pricing': 'pricing',
     'subscribe/:id': 'subscribe',
+    'import_from_slurp': 'importFromSlurp',
```

Comment:
Are you sure we don't want a way to import standalone models? Are most slurps complete sets, or designed to be analyzed only with other slurps?